cpp/include/cudf/concatenate.hpp

```diff
@@ -83,8 +83,8 @@ std::unique_ptr<column> concatenate(
  *
  * @param tables_to_concat host_span of table views to be concatenated into a single table
  * @param mr Device memory resource used to allocate the returned table's device memory
- * @return A single table having all the rows from the elements of `tables_to_concat` respectively
- * in the same order.
+ * @return std::unique_ptr<table> single table having all the rows from the elements of
```

Comment:
Do not include the return type in the `@return` tag:
https://github.com/rapidsai/cudf/blob/branch-22.08/cpp/docs/DOCUMENTATION.md#return